corehq/apps/translations/validator.py

```diff
@@ -158,10 +158,15 @@ def _compare_sheet(self, module_or_form_id, uploaded_rows, for_type):
         :return: diff generated by ghdiff or None
         """
         columns_to_compare = COLUMNS_TO_COMPARE[for_type] + self.lang_cols_to_compare
-        header_indices = {
-            column_name: self._get_current_header_index(module_or_form_id, column_name)
-            for column_name in columns_to_compare
-        }
+        try:
+            header_indices = {
+                column_name: self._get_current_header_index(module_or_form_id, column_name)
+                for column_name in columns_to_compare
+            }
+        except KeyError:
+            raise BulkAppTranslationsException(_(
+                "No {} with Id '{}' is present in the Application.""Please verify your translations file."
```

Comment:
would be nice to stick with how we show such message elsewhere like https://github.com/dimagi/commcare-hq/blob/986410c9c7f7378100f5d299fedb3e4c57469ab8/corehq/apps/app_manager/models.py#L5060.

Reply:
Understood.
Will do that.

Reply:
Done.